website/blog/2018-12-21-state-of-react-native-community.md

```diff
@@ -0,0 +1,51 @@
+authorTwitter: kelset
+category: announcements
+---
+
+This year we made a large number of changes about how we develop and communicate about React Native; we hope that in two years from now we can look back and see that this shift was a turning point for React Native.
+
+A lot of people are excited about the rewrite of React Native's architecture, widely known as [Fabric](https://github.com/react-native-community/discussions-and-proposals/issues/4). Among other things, this will fix fundamental flaws in React Native's architecture and will set up React Native for success in the future together with [JSI and TurboModules](https://github.com/react-native-community/discussions-and-proposals/issues/40).
+
+The biggest shift initiated in 2018 was to empower the React Native Community: from the beginning Facebook encouraged developers from all around the world to participate in React Native's open source project. Since then, a number of core contributors emerged who handle, among other things, the release process.
+
+This year, these members took a few substantial steps towards making the whole community more able to shape the future of this project; two repositories have been created throughout the year precisely with this goal in mind:
+
+## [`react-native-releases`](https://github.com/react-native-community/react-native-releases) 📬
+
+This repository, started around Feb, serves the dual purpose of allowing everyone to keep up the new releases in a more collaborative manner and opened the conversation of what would be part of a certain release to whoever wanted to suggest a cherry-pick (like for [0.57.8](https://github.com/react-native-community/react-native-releases/issues/71) and all its previous versions).
+
+This has been the driving force behind moving away from a monthly release cycle, and the "long term support" approach currently used for version 0.57.x.
+
+Half of the merit in reaching these decisions goes to the other repository created this year:
+
+## [`discussions-and-proposals`](https://github.com/react-native-community/discussions-and-proposals) 🗣
+
+This other ‘meta’ repository, created in July, expanded on the idea of a more open environment for conversations on React Native; while, partially, this need was covered by a few issues labelled [`For Discussion`](https://github.com/facebook/react-native/labels/For%20Discussion) in the main repository, we wanted to give our own spin to the RFC approach that other libraries (like React) have.
+
+This experiment found immediately its role in the React Native lifecycle: even the FB team is now relying heavily on it to discuss what could [be improved in React Native](https://github.com/react-native-community/discussions-and-proposals/issues/64), and coordinate the efforts around the [Lean Core project](https://github.com/react-native-community/discussions-and-proposals/issues/6) - among other interesting conversations.
```

Comment:
```suggestion
This experiment immediately found its role in the React Native lifecycle. The Facebook team is now relying heavily on the RFC process to discuss what could [be improved in React Native](https://github.com/react-native-community/discussions-and-proposals/issues/64), and coordinate the efforts around the [Lean Core project](https://github.com/react-native-community/discussions-and-proposals/issues/6) - among other interesting discussions.
```

- Put the adverb first
- Remove colon
- Remove "even" from "even FB"
- Spell Facebook
- Specify the pronoun because context isn't clear
- Use "discussions" for consistent language